papaparse.js

```diff
@@ -389,6 +390,10 @@
 
 		this.parseChunk = function(chunk)
 		{
+			if (this._chunkIndex == 0 && isFunction(this._config.beforeFirstChunk))
+				chunk = this._config.beforeFirstChunk(chunk);
```

Comment:
In case beforeFirstChunk doesn't return anything, we should still use the chunk as-is. Something like `|| chunk` at the end of the line (except maybe compare against `undefined` so we acknowledge empty strings)